.rultor.yml

```diff
@@ -15,4 +19,4 @@ release:
   script: bash .misc/release.sh
 
 decrypt:
-  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
+  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"  
```

Comment:
see http://stackoverflow.com/a/5813359/3212182 about the trailing newline issue, this one is about trailing spaces right now, before you hadd the missing newline before EOF